dev_guide/authorization.adoc

```diff
@@ -0,0 +1,84 @@
+[[dev-guide-authorization]]
+= Authorization
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+[[checking-if-users-can-create-pods]]
+== Checking if Users Can Create Pods
```

Comment:
@bfallonf Copy/paste fail, sorry. Meant to say I thought title case would be Checking If Users Can Create Pods (capital If). Not a huge deal.